<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>895</width>
    <height>225</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>225</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="GBOX_channel_title">
     <property name="minimumSize">
      <size>
       <width>136</width>
       <height>210</height>
      </size>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,1,0,0,0" columnstretch="2,1,1,1">
        <item row="2" column="0" rowspan="2">
         <widget class="QLabel" name="NON_freq">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>140</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial Black</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Frequency</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QComboBox" name="CBOX_power">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>Vpp</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>dBm</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>mW</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="0" rowspan="2">
         <widget class="QLabel" name="NON_power">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>140</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial Black</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Power</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QComboBox" name="CBOX_phase">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>degree</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>radian</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QDoubleSpinBox" name="SPB_phase_step">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>30</height>
           </size>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-100.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QComboBox" name="CBOX_freq">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>kHz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MHz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>GHz</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QDoubleSpinBox" name="SPB_power_step">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>30</height>
           </size>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-100.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="SPB_freq">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>30</height>
           </size>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="maximum">
           <double>15000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="SPB_power">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>30</height>
           </size>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-80.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QDoubleSpinBox" name="SPB_freq_step">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>30</height>
           </size>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-100.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QDoubleSpinBox" name="SPB_phase">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>30</height>
           </size>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-359.000000000000000</double>
          </property>
          <property name="maximum">
           <double>360.000000000000000</double>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="5" column="1" colspan="3">
         <widget class="QSlider" name="SLB_phase">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="0" rowspan="2">
         <widget class="QLabel" name="NON_phase">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>140</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial Black</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Phase</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="3">
         <widget class="QSlider" name="SLB_power">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="1" colspan="3">
         <widget class="QSlider" name="SLB_freq">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="BTN_oscillo">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial Black</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Oscillo</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BTN_on">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial Black</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>On</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>CBOX_power</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>Form</receiver>
   <slot>changedPowerUnit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>497</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>519</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CBOX_freq</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>Form</receiver>
   <slot>changedFreqUnit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>519</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>519</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CBOX_phase</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>Form</receiver>
   <slot>changedPhaseUnit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>528</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>519</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SPB_power</sender>
   <signal>editingFinished()</signal>
   <receiver>Form</receiver>
   <slot>editedPower()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>1034</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SPB_power_step</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Form</receiver>
   <slot>changedPowerStep()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>741</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>1351</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SPB_freq_step</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Form</receiver>
   <slot>changedFreqStep()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>812</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>1381</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SPB_phase_step</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Form</receiver>
   <slot>changedPhaseStep()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>829</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>1257</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SPB_freq</sender>
   <signal>editingFinished()</signal>
   <receiver>Form</receiver>
   <slot>editedFreq()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>686</x>
     <y>454</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SPB_phase</sender>
   <signal>editingFinished()</signal>
   <receiver>Form</receiver>
   <slot>editedPhase()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>400</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changedFreqUnit()</slot>
  <slot>changedPowerUnit()</slot>
  <slot>changedPhaseUnit()</slot>
  <slot>changedPower()</slot>
  <slot>changedFreq()</slot>
  <slot>changedPhase()</slot>
  <slot>editedPower()</slot>
  <slot>editedFreq()</slot>
  <slot>editedPhase()</slot>
  <slot>changedPowerStep()</slot>
  <slot>changedFreqStep()</slot>
  <slot>changedPhaseStep()</slot>
 </slots>
</ui>
